src/main/java/mods/railcraft/common/items/firestone/ItemFirestoneRefined.java

```diff
@@ -217,7 +217,7 @@ public Entity createEntity(World world, Entity location, ItemStack stack) {
     public void finalizeDefinition() {
         ItemStack ore = EnumOreMagic.FIRESTONE.getStack();
         ItemStack rawfirestone = RailcraftItems.FIRESTONE_RAW.getStack();
-        if (Mod.anyLoaded(Mod.IC2, Mod.IC2_CLASSIC) && RailcraftConfig.getRecipeConfig("ic2.macerator.charcoal")) {
+        if (Mod.anyLoaded(Mod.IC2, Mod.IC2_CLASSIC) && RailcraftConfig.getRecipeConfig("ic2.macerator.ore")) {
```

Comment:
D you actually define this option? I see a "ic2.macerator.ores", but not "ic2.macerator.ore".